src/drivers/rest/data.rs

```diff
@@ -0,0 +1,164 @@
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use lazy_static::lazy_static;
+use mavlink::{self, Message};
+use serde::{Deserialize, Serialize};
+
+lazy_static! {
+    static ref DATA: Data = Data {
+        messages: Arc::new(Mutex::new(MAVLinkVehiclesData::default())),
+    };
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct MAVLinkMessage<T> {
+    pub header: mavlink::MavHeader,
+    pub message: T,
+}
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
```

Comment:
We want to avoid this expensive clone

```suggestion
#[derive(Debug, Default, Deserialize, Serialize)]
```

Reply:
done